cadence/grpc_errors.py

```diff
@@ -0,0 +1,46 @@
+import grpc
+from grpc_status import rpc_status
+
+from cadence.errors import WorkflowExecutionAlreadyStartedError, ServiceBusyError, QueryFailedError, LimitExceededError, \
+    CancellationAlreadyRequestedError, ClientVersionNotSupportedError, DomainAlreadyExistsError, DomainNotActiveError, \
+    EntityNotExistsError, WorkflowExecutionAlreadyCompletedError, FeatureNotEnabledError
+from uber.cadence.api.v1 import error_pb2
+
+
+def process_error(e: grpc.RpcError) -> Exception:
+    status = rpc_status.from_call(e)
+    for detail in status.details:
```

Comment:
There were some errors like `BadRequestError` or `AccessDeniedError` that were not present in the proto file, however I found them in the go client https://github.com/uber-go/cadence-client/blob/master/internal/compatibility/proto/error.go#L36

I'll figure out how to trigger them, how it looks as proto error and I'll map it later. I suspect they might land in a more general category defined by grpc already